src/content/devices/multi/index.html

```diff
@@ -82,12 +82,7 @@ <h2>Local media files:</h2>
 
     </div>
 
-    <p>This demo must be run from localhost or over HTTPS 45.0.2441.x or later.</p>
-      <p><strong>Enable experimental Web Platform features</strong> must be enabled from the chrome://flags page or run Chrome with the following flag:</p>
-      <p>&nbsp;&nbsp;&nbsp;&nbsp;<code>--enable-blink-features=EnumerateDevices,AudioOutputDevices</code></p>
-      <p>More information about running Chrome with flags is available from <a href="https://www.chromium.org/developers/how-tos/run-chromium-with-flags" title="The Chromium Projects: Run Chromium with flags">The Chromium Projects</a>.</p>
-
-      <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/devices/multi" title="View source for this page on GitHub" id="viewSource">View source on GitHub</a>
+    <p>This demo must be run from localhost or over HTTPS Chrome 49 or later, Firefox is not supported yet.</p>
```

Comment:
All the WebRTC demos need to be run from localhost, HTTPS or a `file://` URL. Maybe add a note at the top?

I would remove stuff like 'Chrome 49 or later'. This is no longer relevant.